ls1intum/Artemis
# `General:` Fix inconsistent capitalization of term "Bonus Points" and unify German translation "Bonuspunkte"

### Checklist
#### General



### Motivation and Context


I saw, that the term Bonus Points was not translated correctly in the create/edit view of an exercise.

### Description

1. I changed the German translation to an actual German translation
2. I unified the German translation throughout the many occurrence to "Bonuspunkte" (instead of Bonus Punkte), because Duden says, thats correct.
3. I capitalized the English translation string, because we usually do this (see picture)
4. I also fixed an issue, where the button had a `.` at the end of the label (not related to the bonus point refactoring, but not it's not worth opening a new PR):
<img width="1264" alt="Bildschirmfoto 2021-10-08 um 13 30 46" src="https://user-images.githubusercontent.com/41366522/136583667-0e7d20e5-6a2b-47b7-8412-d819564840cf.png">


### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create an exercise -> is the label correct for German and English
4. Edit an exercise -> is the label correct for German and English
5. Are there any other occurrences of the term, which should be changed?

### Review Progress





#### Manual Tests

### Screenshots


Before German:
<img width="1265" alt="Bildschirmfoto 2021-10-07 um 22 33 32" src="https://user-images.githubusercontent.com/41366522/136523780-2190b056-a156-4fbe-aaa3-ac18de1bc4b6.png">

Before English:
<img width="1243" alt="Bildschirmfoto 2021-10-07 um 22 33 26" src="https://user-images.githubusercontent.com/41366522/136523822-68f34729-bf3a-4d52-a242-acda17d82469.png">
